src/components/entity/__internal__/entity-content-divider.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```

Comment:
can we remove this line to stay consistant to other files?

Reply:
Style consistency is less important here because this PR will no longer be merged into Carbon. Once it is signed-off here it will be handed over to the application developers who can apply their own style or linting rules.